purchases/src/test/java/com/revenuecat/purchases/PurchasesCommonTest.kt

```diff
@@ -68,14 +65,11 @@ import org.junit.runner.RunWith
 import org.robolectric.annotation.Config
 import java.net.URL
 import java.util.Collections.emptyList
-import java.util.concurrent.CountDownLatch
-import java.util.concurrent.TimeUnit
-import kotlin.random.Random
 
 @RunWith(AndroidJUnit4::class)
 @Config(manifest = Config.NONE)
 @Suppress("DEPRECATION")
-internal class PurchasesTest: BasePurchasesTest() {
+internal class PurchasesCommonTest: BasePurchasesTest() {
```

Comment:
I tried to split the tests in "Common" for those that should be run in both flavors, and flavor specific ones.